docs/release/handbook.md

```diff
@@ -0,0 +1,288 @@
+
+### git revision
+
+A git revision is either a git commit, branch, or tag. Ideally people should be providing the manifests repo with tags, to update the manifests from.
+
+
+### Release Candidate (RC)
+
+The Release Candidate (RC) in the manifests repo is always a git **tag**.
+
+The manifests repo will be following the release process below:
+
+1. There will be a **vX.Y-branch** branch, in which the Manifests WG leads will be cherry picking commits on top
+2. Manifests leads will be creating new tags for different RCs, that will be pointing to commits of that release branch
+
+
+## Timeline
+
+
+### Preparation
+
+- [ ] Select a release team
+- [ ] Working groups broadly think about features **with priorities** they want to land for that cycle, have internal discussions, perhaps groom a backlog from previous cycle, get issues triaged, etc.
+- [ ] Ensure members of the Release Team are part of the [release-team](https://github.com/kubeflow/internal-acls/blob/54a454b92e34a98f74a8dc07216dd578f6fa40d5/github-orgs/kubeflow/org.yaml#L950) group
+- [ ] Establish a regular release team meeting as appropriate on the schedule, start off slowly, but meet more often towards the end.
+- [ ] Publish draft schedule to kubeflow-discuss, with actual dates
+- [ ] Get lazy consensus on the release schedule from the WG leads
+- [ ] Ensure schedule also accounts for the point releases AFTER the major release
+
+Criteria for timeline that the team needs to consider
+- Holidays around the world that coincide with members of the release team, WG representatives, and distro representatives.
+- Enterprise budgeting/approval lifecycle. (aka users have their own usage and purchase requirements and deadlines)
+- Kubecon dates - let’s not hard block on events, but keep them in mind since we know community members might get doublebooked.
+- Associated events (aka. AI Day at Kubecon, Tensorflow events) - we want to keep them in mind.
+
+**Success Criteria:** Release team selected, schedule sent to kubeflow-discuss, all release team members have the proper permissions and are meeting regularly.
+
+
+### Development (10 weeks)
+
+Normal development in the different WGs and in the <https://github.com/kubeflow/manifests> repo.
+
+**Success Criteria:**
+* (Optional but encouraged): Issues tracking new features should also provide information on whether the docs should be updated for that feature
+
+
+### Feature Freeze (2 weeks)
+
+From that phase and forward updates to the manifests repo must only be fixing component bugs. No new commits, in the manifests repo, that update a component to include a new feature are allowed.
+
+**Actions for other WGs:**
+
+- Provide a _git revision_ which the Manifests leads will use to update the files in the manifests repo. WGs are highly encouraged to follow the release process described above
+- Provide an initial list of issues that the WG would like to close until the final release
+- Work on closing the provided opened issues
```

Comment:
s/opened/open
good to also add a milestone tag to the issue? e.g., release 1.3.1 or 1.4 type thing for quick retrieval and tracking of progress.

Reply:
A KanBan would be an abstraction on top. At the bare minimum the WGs will need to provide the issues.

But if the issues are also organized in a KanBan board then even better! I'll add a comment that having a KanBan board can help even more to keep track of the issues

Reply:
Added a commit